### PR TITLE
Refactor fetch operations, API IP, build information

### PR DESCRIPTION
**Pull request by maxitect**
* Rename playlist.ts to playlistTypes.ts for clarity
* Added UserInputs type to playlistTypes.ts to be consistent with API project types
* Deleted unused genres.json from utils folder as this is duplicated from the data folder which is the one being used
* Updated import paths using playlist.ts to playlistTypes.ts in Track.tsx
* Updated import paths using playlist.ts to playlistTypes.ts in Tracklist.tsx
* Updated import paths using playlist.ts to playlistTypes.ts in PlaylistPage.tsx
* Commented out loading page import in Content.tsx as this is unused and causes issues when building for deployment
* For consistency with API naming:
  * Rename 'feel' parameter to 'eventDescription' in userResponses object and subsequent input component inside InputPage.tsx
  * Rename 'genre' parameter to 'musicGenre' in userResponses object and subsequent input component inside InputPage.tsx
  * Rename 'quant' parameter to 'playlistCount' in userResponses object and subsequent input component inside InputPage.tsx
* Move callAPI function out of InputPage.tsx into a utils folder renamed to fetchPlaylist.tsx
* Update InputPage.tsx to import fetchPlaylist.tsx and call it inside useEffect hook
* Build config file updated to include all files required in build

### CODE REVIEW

File: src/playlist/Tracklist.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // Tracklist.tsx
 import Track from './Track';
-import { TrackProps, TracklistProps } from '../utils/playlist';
+import { TrackProps, TracklistProps } from '../utils/playlistTypes';
 
 function Tracklist({ playlist }: { playlist: TracklistProps }) {  // Fix prop structure
```
Comment (JasonWarrenUK) [inline src/playlist/Tracklist.tsx:5]: I'll leave it unmerged til you're done with this?

File: src/sections/Content.tsx [hunk 2]
```diff
@@ -2,7 +2,7 @@ import { createContext, useState } from 'react';
 import DevTools from './DevTools';
 import LandingPage from '../pages/LandingPage';
 import InputPage from '../pages/InputPage';
-import LoadingPage from '../pages/LoadingPage';
+//import LoadingPage from '../pages/LoadingPage';
```
Comment (JasonWarrenUK) [inline src/sections/Content.tsx:5]: BIN THIS

File: src/playlist/Tracklist.tsx (hunk 1 above)
Comment (maxitect) [reply src/playlist/Tracklist.tsx]: Unnecessary comments removed, thanks!

File: src/sections/Content.tsx (hunk 2 above)
Comment (maxitect) [reply src/sections/Content.tsx]: It is BINNED!

**Review by Gaajia — APPROVED** [review]

**Pull request merged**